atlp-rwanda/e-comm-team-amigos-bn
# #184581188 A User Should be able to list all Items

#### What does this PR do?
This PR help A user/Seller to get a list of product in his/her collection

#### Description of Task to be completed?
- Creating route to get items in seller collection
- protecting route
- testing route
- document route

#### How should this be manually tested?
- Clone this repo git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn
- Switch your working directory cd e-comm-team-amigos-bn
- Create an .env file and update it with .env-example
- Install all the dependencies by running npm install
- make sure you have testing tool like `Postman`
- Then run the project by typing `npm run dev`
- you should create a user with a role of `vendor`

#### Endpoints to be tested:
- `/user/login/`
- `/user/otp/`
- `/product/create/`
- `/product/collection/`

#### Any background context you want to provide?
- Have Node Js installed in your local machine

#### What are the relevant pivotal tracker/Trello stories?

Story ID: [#184581188](https://www.pivotaltracker.com/story/show/184581188)

#### Screenshots
#### All Test are passing
![Screenshot (142)](https://user-images.githubusercontent.com/79166151/226837993-3b273583-dc8a-4013-b854-3bd3299771dd.png)

#### Endpoint in documentation
![Screenshot (141)](https://user-images.githubusercontent.com/79166151/226838015-e5170380-41ef-40c9-8fd9-9afe7c2eddfc.png)

#### Questions:
